packages/arb-token-bridge-ui/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "3.3.0",
   "license": "Apache-2.0",
   "private": true,
+  "type": "module",
```

Comment:
https://docs.synpress.io/docs/getting-started#installation
> Synpress is now an ESM-only package. Don’t use require() to import it, and make sure your nearest package.json contains "type": "module".

This is the reason why we change so many other files regarding .ts .cjs .js .mjs and the generation script commands